modules/core/include/visp3/core/vpCameraParameters.h

```diff
@@ -229,6 +232,28 @@
 
   An XML parser for camera parameters is also provided in vpXmlParserCamera.
 
+  <b>II. Kannala-Brandt camera model</b>
```

Comment:
Probably `vpXmlParserCamera`, `vpPixelMeterConversion` and `vpMeterPixelConversion` should be updated, at least to issue an exception if the Kannala-Brandt model is not supported.

Reply:
Work in progress not commited yet